h2/connection.py

```diff
@@ -1015,8 +1018,6 @@ def reset_stream(self, stream_id, error_code=0):
         stream = self._get_stream_by_id(stream_id)
         frames = stream.reset_stream(error_code)
         self._prepare_for_sending(frames)
-        self._reset_streams.add(stream_id)
-        del self.streams[stream_id]
```

Comment:
Why did this get removed? I'm not opposed to removing it, I just want to understand why.